### ISSUE TITLE
Feature: Do not consider a complete (but not completed) anima quest as an anima quest

### ISSUE THREAD

**Issue by neocryptek** [TLDRMissions/TLDRMissions#616]
**TLDR Version of your idea**
TLDR version: 10.1-008
Do not consider a complete (but not completed) anima quest as an anima quest.

**Slightly longer version**
If a player has an anima quest in their quest log, but it is complete: do not consider it as having an anima quest for the setting to only complete anima quests if a player has an active anima quest. 

**Additional context**
If a player has an anima quest that is complete but still in their quest log, the addon will still finish anima missions, wasting the anima (for next week's quest). This way the player does not need to turn in the anima quest right away for the setting to work as expected.
Small tweak to logic added in #613 to check if the quest is complete (C_QuestLog.IsComplete) in the existing check to see if it is in their quest log.

Thank you! This feature has saved me a ton of time as I've been doing the same manually for a while now.

**Comment by teelolws** [comment]
Okay this should do it. I don't have the means to test it so lmk if it doesn't work.

**Comment by gizzmo** [comment]
Small problem with this change. I just turned in the quest, as its finished, but can not complete the anima missions. If I were to finish the quest early in the week, than my followers are stuck, and not able to do any other missions. (unless I turn off the option)

Tested with the version 10.1.008, and I see that it also has the same problem. With a completed and turned-in quest, missions cant be completed.

**Comment by teelolws** [comment]
Well thats the point of this option. It stops you *completing* the anima missions unless they'll give you credit towards the weekly quest. Now you can turn in the anima missions the following week instead.

If you want them to turn in the anima anyway then it sounds like this option is not for you.

**Comment by gizzmo** [comment]
I assumed the option was protection from completing missions before acquiring the quest on reset, making you lose out on that progress for the week. 

So if I have a great week on missions (or do other stuff that awards anima) and I finish the quest (either turned in or not) on Wednesday. I'm unable to finish any anima mission, and my followers are lock, unless I disable the option. (Running the risk of forgetting to turn it back on before/on reset) Making all anima missions **only** for the quest and not being able to continue to far farm more after the quest.

**Comment by teelolws** [comment]
Thats what the original request was https://github.com/TLDRMissions/TLDRMissions/issues/611 : they wanted it to block completing anima missions unless the quest was in the log and incomplete!

I'll consider adding a dropdown next to the option where you can disable "not yet collected" / "completed in log" / "completed this week". That will be a job for another day though.

**Comment by teelolws** [comment]
Option added:
![image](https://github.com/TLDRMissions/TLDRMissions/assets/49792789/dd299d48-4d6f-4524-818f-906ee257ccf7)
